dataflow/src/org/checkerframework/dataflow/cfg/CFGBuilder.java

```diff
@@ -1518,6 +1518,9 @@ public PhaseOneResult process(
                 TreeBuilder treeBuilder,
                 AnnotationProvider annotationProvider) {
             this.env = env;
+            if (trees == null) {
+                trees = Trees.instance(env);
+            }
```

Comment:
Ignore this. Not sure why rebasing didn't get rid of it, will fix it before final PR.